app/presenters/booking_presenter.rb

```diff
@@ -0,0 +1,15 @@
+class BookingPresenter < SimpleDelegator
+
+  def self.paypal_url(root_path, id, cost, passenger)
+
+    values = {
+        business: "merchant@ruth-flyapp.com",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.